pkg/collector/python/tagger.go

```diff
@@ -36,7 +36,16 @@ func Tags(id *C.char, cardinality C.int) **C.char {
 	goID := C.GoString(id)
 	var tags []string
 
-	tags, _ = checkContext.tagger.LegacyTag(goID, types.TagCardinality(cardinality))
+	// Generate EntityID from the entity ID string.
+	// This is done for backward compatibility with the Python checks as the Tags() signature cannot be changed.
```

Comment:
looks good!
maybe you could add a comment to explain that you have considered the performance aspect of this change, and why it is not anymore a blocker